bitrise.yml

```diff
@@ -0,0 +1,172 @@
+---
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+workflows:
+  DeployFirebase:
+    description: |
+      The workflow builds and deploys the app in Firebase and Bitrise step.
+
+      For testing the *retry_on_failure* test repetition mode is enabled.
+
+      Next steps:
+      - Set up [Connecting to an Apple service with API key](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html##).
+      - Or further customise code signing following our [iOS code signing](https://devcenter.bitrise.io/en/code-signing/ios-code-signing.html) guide.
+    steps:
+    - authenticate-host-with-netrc@0:
+        inputs:
+        - username: "$GITHUB_USER_NAME"
+        - password: "$GITHUB_PASSWORD"
+        - host: github.com
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - fastlane-match@0:
+        title: Fastlane Match Development
+        inputs:
+        - git_branch: "$FASTLANE_MATCH_GIT_BRANCH"
+        - app_id: "$APP_BUNDLE_ID"
+        - team_id: "$BITRISE_APLLE_DEV_TEAM_ID"
+        - decrypt_password: "$FASTLANE_MATCH_PASSWORD"
+        - git_url: "$FASTLANE_MATCH_GIT_URL"
+    - fastlane-match@0:
+        title: Fastlane Match Adhoc
+        inputs:
+        - git_branch: "$FASTLANE_MATCH_GIT_BRANCH"
+        - app_id: "$APP_BUNDLE_ID"
+        - decrypt_password: "$FASTLANE_MATCH_PASSWORD"
+        - type: adhoc
+        - team_id: "$BITRISE_APLLE_DEV_TEAM_ID"
+        - git_url: "$FASTLANE_MATCH_GIT_URL"
+    - fastlane@3:
+        inputs:
+        - work_dir: "./fastlane"
+        - update_fastlane: 'false'
+        - apple_id: "$APPLE_ID_TAHER"
+        - password: "$APPLE_ID_TAHER_PASS"
+        - app_password: "$APPLE_TAHER_APP_SPECIFIC_PASS"
+        - lane: "$FASTLANE_LANE_TO_RUN"
+    - cache-push@2: {}
+    - firebase-app-distribution@0:
+        inputs:
+        - app: 1:999217461241:ios:7f30e2afa1e2f45e6f6f28
```

Comment:
I think it's better to put this in Secret.

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/24/commits/f929d2482b81136e10768dcebc953b3d50315e5e)